Framework/API/src/Column.cpp

```diff
@@ -5,7 +5,6 @@
 //     & Institut Laue - Langevin
 // SPDX - License - Identifier: GPL - 3.0 +
 #include <algorithm>
-#include <iostream>
```

Comment:
I think the OSX compiler error indicates that this requires both `<istream>` and `<ostream>` for the `<<` and `>>` operators.

Reply:
hmm, odd that it only complains about it on osx, but I'll add it back in I guess.